clang/test/SemaCXX/lambda-expressions.cpp

```diff
@@ -194,6 +194,11 @@ namespace ModifyingCapture {
     [=] {
       n = 1; // expected-error {{cannot assign to a variable captured by copy in a non-mutable lambda}}
     };
+    const int cn = 0;
```

Comment:
I think we should have also tested a `constexpr` variable case and maybe a `auto &` assigned from `cn` as well just to cover a more complete test set.